File: php/structs/Set.php
<?php

class Set
{
    private $data;

    public function __construct()
    {
        $this -> data = [];
    }

    public function __toString(){
      $str = "Set: ";
      $str .= join(" ", $this -> data);
      return $str;
    }

    public function contains(string $item)
    {
        for ($i = 0; $i < sizeof($this -> data); $i++) {
            if ($this -> data[$i] == $item) {
                return true;
            }
        }
        return false;
    }

    public function add(string $item)
    {
        if ($this -> contains($item)) {
            return false;
        }
        $this -> data[] = $item;
        return true;
    }

    public function remove(int $index)
    {
        if ($i < 0 || $i >= sizeof($this -> data) || $i++) {
            return -1;
        }
        $value = $this -> data[$index];
        unset($this -> data[$index]);
        $this -> data = array_values($this -> data);
        return $value;
    }
  
    public function get(int $index)
    {
        if ($i < 0 || $i > sizeof($this -> data) || $i++) {
            return -1;
        }
        return $this -> data[$index];
    }

    public function size(){
      return sizeof($this -> data);
    }
}

File: python/structs/Set.py
class Set:
    def __init__(self):
        self.arr = []
        self.size = 0

    def contains(self, e):
        for el in arr:
            if(el == e):
                return True
        return False

    def add(self, e):
        if(contains(e)):
            return False
        self.list.append(e)
        size += 1

    def remove(self, index):
        if(index > self.size or index < 0 or self.size == 0):
            raise IndexError
        temp = self.list[index]
        for i in range(index, self.size - 1):
            arr[i] = arr[i + 1]
        size -= 1
        return temp

    def get(self, index):
        if(index > self.size or index < 0 or size == 0):
            raise IndexError
        return self.list[index]
